internal/cli/kraft/cloud/instance/create/create.go

```diff
@@ -743,6 +743,10 @@ func (opts *CreateOptions) Pre(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("could not populate metro and token: %w", err)
 	}
 
+	if opts.ScaleToZeroCooldown < time.Millisecond && opts.ScaleToZeroCooldown != 0 {
+		return fmt.Errorf("scale-to-zero-cooldown needs to be at least 1ms: %s", opts.ScaleToZeroCooldown)
```

Comment:
That is no ideal, because there is a configurable lowest value. So if people try 100us, they get the error to use 1ms. Then they use 1ms and get an error that Xms is the minimum.

Reply:
Yes, but I think the lowest it can be is still 1ms, so it's good to at least catch that

If the actual limit is higher they'll just get another error telling them the actual limit